lua/nvim-lsp-installer/servers/slint_lsp/init.lua

```diff
@@ -0,0 +1,36 @@
+local server = require "nvim-lsp-installer.server"
+local Data = require "nvim-lsp-installer.data"
+local std = require "nvim-lsp-installer.installers.std"
+local platform = require "nvim-lsp-installer.platform"
+local process = require "nvim-lsp-installer.process"
+local context = require "nvim-lsp-installer.installers.context"
+
+local coalesce, when = Data.coalesce, Data.when
+
+return function(name, root_dir)
+    local archive_name = coalesce(
+        when(platform.is_linux and platform.arch == "x64", "slint-lsp-linux.tar.gz"),
+        when(platform.is_win and platform.arch == "x64", "slint-lsp-windows.zip")
+    )
+    return server.Server:new {
+        name = name,
+        root_dir = root_dir,
+        homepage = "https://slint-ui.com/",
+        languages = { "slint" },
+        installer = {
+            context.use_github_release_file("slint-ui/slint", archive_name),
+            context.capture(function(ctx)
+                if platform.is_win then
+                    return std.unzip_remote(ctx.github_release_file)
+                else
+                    return std.untargz_remote(ctx.github_release_file)
+                end
+            end),
+        },
+        default_options = {
+            cmd_env = {
+                PATH = process.extend_path { root_dir, "slint-lsp" },
+            },
```

Comment:
By extending the PATH environment used when starting the server, we can avoid having to override the `cmd` set by lspconfig.

Reply:
Hey, thanks for fixing this!